metabrainz/templates/users/profile.html

```diff
@@ -94,6 +56,49 @@ <h1 class="page-title">Your Profile</h1>
     </div>
   </div>
 
+  {% if current_user.is_commercial %}
+    {% if current_user.state == 'active' and current_user.good_standing %}
```

Comment:
Use the same block for active and limited. Both types of users should see the same message. Limited users should not see the "generate access token" field below.